cloudigrade/util/aws/helper.py

```diff
@@ -88,7 +88,7 @@ def _handle_dry_run_response_exception(action, e):
     raise e
 
 
-def _verify_policy_action(session, action):
+def _verify_policy_action(session, action):  # noqa: C901
```

Comment:
The `noqa: C901` is unfortunate but understandable since this function is a massive list of `elif`s. Would you add a note in the docstring explaining what `C901` means and that we're not just recklessly ignoring the check?

Reply:
Yes, that's a good idea.